radix-transactions/src/manifest/static_resource_movements/effect.rs

```diff
@@ -0,0 +1,1541 @@
+                            }
+                            AccountMethod::TryDepositOrAbort(input) => {
+                                input.output(node_id, inputs, instruction_index)
+                            }
+                            AccountMethod::Burn(input) => input.output(node_id, inputs, instruction_index),
+                            AccountMethod::BurnNonFungibles(input) => {
+                                input.output(node_id, inputs, instruction_index)
+                            }
+                            AccountMethod::AddAuthorizedDepositor(input) => {
+                                input.output(node_id, inputs, instruction_index)
+                            }
+                            AccountMethod::RemoveAuthorizedDepositor(input) => {
+                                input.output(node_id, inputs, instruction_index)
+                            }
+                            AccountMethod::TryDepositOrRefund(input) => {
+                                input.output(node_id, inputs, instruction_index)
+                            }
+                            // TODO: A bit of a hack to support expressions. We need a better way to
+                            // do this in the future, but certainly not one for this PR.
+                            AccountMethod::DepositBatch(..) => AccountDepositBatchManifestInput {
+                                buckets: Default::default(),
+                            }
+                            .output(node_id, inputs, instruction_index),
+                            AccountMethod::TryDepositBatchOrRefund(..) => {
+                                AccountTryDepositBatchOrRefundManifestInput {
+                                    buckets: Default::default(),
+                                    authorized_depositor_badge: None,
+                                }
+                                .output(node_id, inputs, instruction_index)
+                            }
+                            AccountMethod::TryDepositBatchOrAbort(..) => {
+                                AccountTryDepositBatchOrAbortManifestInput {
+                                    buckets: Default::default(),
+                                    authorized_depositor_badge: None,
+                                }
+                                .output(node_id, inputs, instruction_index)
+                            }
+                        },
+                    }
+                }
+            },
+            TypedNativeInvocation::ConsensusManagerPackage(consensus_manager_invocations) => {
+                match consensus_manager_invocations {
+                    ConsensusManagerInvocations::ValidatorBlueprint(validator_blueprint_invocations) => {
+                        match validator_blueprint_invocations {
+                            ValidatorBlueprintInvocations::Function(..) => {
+                                unreachable!()
```

Comment:
(Removed in https://github.com/radixdlt/radixdlt-scrypto/pull/1931)

Reply:
It's because there are no validator functions.

Reply:
Ah right, well it's better to have the compiler verify (via an empty match) this where possible to avoid a panic if this changes in future 👍. I matched on an empty *self in #1391 to avoid this panic.